### DISCUSSION TITLE
Gateway stops sending data, and WebUI is not available

### DISCUSSION THREAD

**Discussion by MarkusMit** [emsesp/EMS-ESP32#1962] [General]
Hi,
today it happened the second time in ten days that the Gateway stopped sending data (to Home Assistant). I usually only notice after a few hours, when the data stops changing.
When I then try to connect to the WebUI/Dashboard I also only get an "not available" error from thebrowser.

After unplugging and repluging the ServiceJack-connector the Gateway starts working just fine again.

Should it happen again, Is there anything I can check before or after restarting the gateway, to get a hint what the problem could be? (Are logs preserved somewhere?)

(EMS-ESP Version `3.7.0-dev.26 (ESP32-S3)`, Connected to CS6800i/WLW176i via Service Jack.)

**Comment by MichaelDvP** [comment]
A possible reason is in mesh network the connection break and the esp reconnect to a far away mesh-point with very low data rate. Then you get timeouts in web/mqtt. Try to force connection to the near meshpoint by setting the BSSID.

**Reply by MarkusMit** [reply]
Hi Michael, 
thanks for the hint, I'll give it a try. 

But this shouldn't break the availability of the EMS-ESP WebUI/Dashboard, should it?

**Reply by proddy** [reply]
https://emsesp.org/Troubleshooting/#ems-esp-freezes

**Reply by MarkusMit** [reply]
Thanks.
